Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Core/SendScriptEvent.cpp

```diff
@@ -343,7 +343,7 @@ namespace ScriptCanvas
                         {
                             Data::Type outputType(AZ::BehaviorContextHelper::IsStringParameter(*result) ? Data::Type::String() : Data::FromAZType(result->m_typeId));
                             // multiple outs will need out value names
-                            const AZStd::string resultSlotName(AZStd::string::format("Result: %s", Data::GetName(outputType).c_str()));
+                            const AZStd::string resultSlotName(Data::GetName(outputType).c_str());
```

Comment:
repeat: the `.c_str()` isn't required

```suggestion
                            const AZStd::string resultSlotName(Data::GetName(outputType));
```